src/components/masthead/masthead.js

```diff
@@ -86,6 +92,16 @@ class Masthead extends React.Component {
       <React.Fragment>
         <Toolbar>
           <ToolbarGroup className={css(accessibleStyles.screenReader, accessibleStyles.visibleOnLg)}>
+            <ToolbarItem className={css(accessibleStyles.screenReader, accessibleStyles.visibleOnMd)}>
+              <Button
+                className="pf-c-button pf-m-plain"
+                aria-label="Settings"
+                variant="plain"
+                onClick={this.onSettingsClick}
+              >
+                <i className="fas fa-cog" aria-hidden="true" />
```

Comment:
I think there is a React Icon for this, but we can update in a future story.

Reply:
yeah you can import the Cog icon
```suggestion
                < CogIcon/>
```

then just add an import above as 

```suggestion
                import { CogIcon } from '@patternfly/react-icons';
```

Reply:
I imported the CogIcon from react-icons and it looks good. Thanks.